boltview/array_view.h

```diff
@@ -148,12 +148,7 @@ class DeviceArrayView: public DeviceArrayConstView<TElement> {
 	DeviceArrayView &operator=(const DeviceArrayView &) = default;
 
 	BOLT_DECL_DEVICE
-	Element operator[](IndexType index) const {
```

Comment:
I believe that this operator[] was unnecessary. Moreover, it caused problems during compilations, so I removed it.